File: local/HDR__/c_src/oddEven.c
#include<stdio.h>
int main(void)
{
	int num;
	printf("input num : ");
	scanf("%d", &num);
	
	int isOdd;
	isOdd = (num % 2 == 1);
		
	printf("num : %d --- isOdd : %d\n", num, isOdd);
	return 0;
}

File: local/HDR__/python/oddEven.py
num = int(input("num : "))

if num%2 == 1:
    print("num is odd number")
else :
    print("nums is even number")
    
result = [num**2 for num in range(1,10+1) if num % 2 == 1]

print(result)    




    